entity-services/src/main/xdmp/entity-services/entity-services-codegen.xqy

```diff
@@ -197,50 +192,36 @@ declare function {$prefix}:extract-instance-{$entity-type-name}(
     $source as node()?
```

Comment:
I know this was already `$source as node()?` but I wanted to point out that json:object() and json:array() are not node(). They are item(). So this means you can't pass in json to this generated function. For the DHF I changed it to item()? to be more accepting of others.